src/JustEat.StatsD/Buffered/IStatsDBufferedTransport.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace JustEat.StatsD.Buffered
```

Comment:
I think this would be better in `JustEat.StatsD` rather than `JustEat.StatsD.Buffered` to make it more discoverable.